private field 'count' is unused in ThreadPool

```
./thirdparty/vhacd/public/VHACD.h:5698:9: warning: private field 'count' is not used [-Wunused-private-field]
 5698 |     int count;
      |         ^
```
I noticed this warning when I tried using it.

--- a/include/VHACD.h
+++ b/include/VHACD.h
@@ -5695,7 +5695,6 @@ class ThreadPool {
     std::mutex task_mutex;
     std::condition_variable cv;
     bool closed;
-    int count;
 };
 
 ThreadPool::ThreadPool()
@@ -5705,7 +5704,6 @@ ThreadPool::ThreadPool()
 
 ThreadPool::ThreadPool(int worker)
     : closed(false)
-    , count(0)
 {
     workers.reserve(worker);
     for(int i=0; i<worker; i++) 

